src/main/java/com/android/volley/AsyncCache.java

```diff
@@ -0,0 +1,82 @@
+package com.android.volley;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.atomic.AtomicReference;
+
+public abstract class AsyncCache implements Cache {
+
+    public interface OnGetCompleteCallback {
+        void onGetComplete(Entry entry);
```

Comment:
Since `entry` can be null, please annotate the parameter as `@Nullable`, so whoever implements the class will be aware that the method parameter can be null.

Reply:
fixed